Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/hashtablesransomnote/Result.java

```diff
@@ -1,14 +1,15 @@
-package hackerrank.hashtableransomnote;
+package hackerrank.hashtablesransomnote;
 
 import java.util.HashMap;
+import java.util.Map;
 import java.util.List;
 
-class Result {
+public class Result {
 
     public static void checkMagazine(List<String> magazine, List<String> note) {
 
-        HashMap<String, Integer> wordsFreqMagazine = createWordsFrequencyTable(magazine);
-        HashMap<String, Integer> wordsFreqNote = createWordsFrequencyTable(note);
+        Map<String, Integer> wordsFreqMagazine = createWordsFrequencyTable(magazine);
+        Map<String, Integer> wordsFreqNote = createWordsFrequencyTable(note);
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...